package.json

```diff
@@ -36,26 +36,26 @@
     "stop:blockchain:client": "bash ./scripts/stop-blockchain-client.sh",
     "pretest:contracts": "pnpm run stop:blockchain:client",
     "pretest:contracts:extensions": "pnpm run stop:blockchain:client",
-    "test:contracts": "pnpm run start:blockchain:client && npx hardhat test ./test/contracts-network/* ./test/packages/* --network development",
-    "test:contracts:extensions": "pnpm run start:blockchain:client && npx hardhat test ./test/extensions/* --network development",
+    "test:contracts": "npx hardhat test ./test/contracts-network/* ./test/packages/*",
+    "test:contracts:extensions": "npx hardhat test ./test/extensions/*",
     "test:contracts:upgrade:parity": "pnpm run clean:test:contracts | pnpm run start:blockchain:client parity & pnpm run generate:test:contracts && npx hardhat test ./test/upgrade/* --network integration",
-    "test:contracts:upgrade:ganache": "pnpm run clean:test:contracts | pnpm run start:blockchain:client ganache & pnpm run generate:test:contracts && npx hardhat test ./test/upgrade/* --network development",
-    "test:contracts:gas": "pnpm run start:blockchain:client && npx hardhat test test/misc/gas-costs.js --network development",
-    "test:contracts:patricia": "pnpm run start:blockchain:client ganache && npx hardhat test packages/reputation-miner/patricia-test.js --network integration",
-    "test:contracts:bridging:1": "CHAIN_ID_1=265669100 CHAIN_ID_2=265669101 pnpm run start:blockchain:client:both && HARDHAT_FOREIGN=false MINING_CHAIN_ID=265669100 npx hardhat test ./test/cross-chain/* --network development",
-    "test:contracts:bridging:2": "CHAIN_ID_1=265669101 CHAIN_ID_2=265669100 pnpm run start:blockchain:client:both && HARDHAT_FOREIGN=true MINING_CHAIN_ID=265669100 CHAIN_ID=265669101 npx hardhat test ./test/cross-chain/* --network development",
-    "test:contracts:chainid": "pnpm run start:blockchain:client && npx hardhat test ./test/misc/chainid.js",
-    "test:contracts:e2e": "pnpm run start:blockchain:client && npx hardhat test test/misc/end-to-end.js --network development",
+    "test:contracts:upgrade:ganache": "pnpm run clean:test:contracts | pnpm run start:blockchain:client ganache & pnpm run generate:test:contracts && npx hardhat test ./test/upgrade/* --network integration",
+    "test:contracts:gas": "npx hardhat test test/misc/gas-costs.js",
+    "test:contracts:patricia": "pnpm run start:blockchain:client ganache && hardhat test packages/reputation-miner/patricia-test.js --network integration",
```

Comment:
Why ditch the `npx` here?

Reply:
Don't remember it being deliberate, not sure what happened. Have restored.